itests/camel-k-itests-loader-groovy/src/main/java/org/apache/camel/k/loader/yaml/quarkus/Application.java

```diff
@@ -37,7 +37,7 @@
 
     @POST
     @Path("/load-routes/{name}")
-    @Consume(MediaType.TEXT_PLAIN)
```

Comment:
Hey @lburgazzoli, would you mind a review about this part, please? I am not entirely sure my understanding is correct here, but I assume we need `javax.ws.rs.Consumes` instead of `org.apache.camel.Consume` as this could lead to bogus setup of our own routes w/ Camel Quarkus 2.0. 

In other words, it would cause our routes to be setup as `text/plain` causing the failure:

```
Caused by: java.lang.RuntimeException: org.apache.camel.FailedToCreateRouteException: Failed to create route route1: Route(route1)[From[text/plain] -> [Bean[ref:application-1812... because of No endpoint could be found for: text/plain, please check your classpath contains the needed Camel component jar.
```

Reply:
Damn, I think I made a mistake, there are tooo many consume/produce annotations so yes, we need `javax.ws.rs.Consumes`

Reply:
Thanks for review!